spec/datetimeformat.html

```diff
@@ -303,6 +317,15 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"fractionalSecondDigits"`, then
+            1. If _fractionalSecondDigits_ is greater than 0, then
+              1. Let _v_ be _tm_.[[millisecond]].
+              1. If _fractionalSecondDigits_ = 1, then
```

Comment:
You can say

```
Let _v_ be _v_ × 10<sup>-_fractionalSecondDigits_</sup>
```

See https://github.com/tc39/proposal-unified-intl-numberformat/blob/master/section11/numberformat_proposed.html#L631

Reply:
Or whatever the math is (I think I'm off by one).  The main thing is that you can use power of 10 so you don't need the conditional statement.  It makes it more generalizable when we eventually extend this to support nanoseconds.